jsx/Form.js

```diff
@@ -416,6 +456,7 @@ SelectElement.defaultProps = {
   multiple: false,
   disabled: false,
   required: false,
+  sortByValue: false,
```

Comment:
I'm thinking it may actually be useful to have this be default behaviour, thoughts?